Source/Dafny/Compilers/SinglePassCompiler.cs

```diff
@@ -58,19 +50,24 @@ public abstract class Compiler {
       }
       return n;
     }
+    
+    public CompilerFactory Factory { get; }
 
     public ErrorReporter Reporter;
 
     public CoverageInstrumenter Coverage;
 
-    protected void Error(Bpl.IToken tok, string msg, ConcreteSyntaxTree/*?*/ wr, params object[] args) {
+    protected static void ReportError(ErrorReporter reporter, Bpl.IToken tok, string msg, ConcreteSyntaxTree/*?*/ wr, params object[] args) {
       Contract.Requires(msg != null);
       Contract.Requires(args != null);
 
-      Reporter.Error(MessageSource.Compiler, tok, msg, args);
-      if (wr != null) {
-        wr.WriteLine("/* {0} */", string.Format("Compilation error: " + msg, args));
-      }
+      reporter.Error(MessageSource.Compiler, tok, msg, args);
+      // FIXME this is not valid syntax in all supported languages (e.g. Python)
```

Comment:
FIXME. Please replace this either with a fix or a GH issue.

Reply:
https://github.com/dafny-lang/dafny/issues/1729